src/plugins/curlget/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 if (DEPENDENCY_PHASE)
 	find_package(CURL)
-
 	if(NOT CURL_FOUND)
 		remove_plugin(curlget "Curl-dev not found")
 	endif ()
+	find_package(OpenSSL)
```

Comment:
Same pattern: you have to make sure that only a single variable is passed to every argument of `addPlugin`. Or any problems by doing so?